pymc/gp/gp.py

```diff
@@ -148,18 +148,23 @@ class Latent(Base):
     def __init__(self, *, mean_func=Zero(), cov_func=Constant(0.0)):
         super().__init__(mean_func=mean_func, cov_func=cov_func)
 
-    def _build_prior(self, name, X, reparameterize=True, jitter=JITTER_DEFAULT, **kwargs):
+    def _build_prior(
+        self, name, X, num_outputs=1, reparameterize=True, jitter=JITTER_DEFAULT, **kwargs
+    ):
         mu = self.mean_func(X)
         cov = stabilize(self.cov_func(X), jitter)
         if reparameterize:
             size = np.shape(X)[0]
             v = pm.Normal(name + "_rotated_", mu=0.0, sigma=1.0, size=size, **kwargs)
-            f = pm.Deterministic(name, mu + cholesky(cov).dot(v), dims=kwargs.get("dims", None))
+            f_single = pm.Deterministic(
+                name, mu + cholesky(cov).dot(v), dims=kwargs.get("dims", None)
+            )
         else:
-            f = pm.MvNormal(name, mu=mu, cov=cov, **kwargs)
+            f_single = pm.MvNormal(name, mu=mu, cov=cov, **kwargs)
+        f = pt.stack([f_single] * num_outputs, axis=0) if num_outputs > 1 else f_single
```

Comment:
This approach won't wont, because what you're doing here is repeating a single GP, so each output is the same.  We want each output to be as if they were independent draws from the same GP.